Localization/MobileCenterDistribute/ko-KR/MobileCenterDistribute.strings

```diff
@@ -0,0 +1,15 @@
+/*
+ * Update dialog buttons
+ */
+"MSDistributeUpdateNow" = "지금 업데이트";
+"MSDistributeAskMeInADay" = "하루 안에 물어보기";
```

Comment:
I'm pretty sure this is wrong. 😢

Reply:
This should be `하루동안 열지 않기`.

Reply:
I can replace this one, but they're all auto-generated.  Our team settings right now aren't set for human translation

Reply:
Actually, I can replace it, but whenever changes are detected by the script it'll be overwritten with the bad version... Maybe we should ask for human translation?  The cost on that is very high though, I don't know what the SLA on human translation is...

Reply:
That's why I commented but approved it. I don't this manual change is a good idea because it will be changed every time we submit translation to TouchDown.